src/js/tools/drawing/Stroke.js

```diff
@@ -97,8 +97,35 @@
       linePixels = pskl.PixelUtils.getLinePixels(col, this.startCol, row, this.startRow);
     }
 
-    pskl.PixelUtils.resizePixels(linePixels, penSize).forEach(function (point) {
```

Comment:
Here, I exchange the square-style resize for a cross-shape resize in lines 124-129.